extension/manifest.json

```
@@ -3,7 +3,7 @@
   "manifest_version": 2,
   "name": "iVelt Forum Helper",
   "description": "Adds accessibility features, and other usability improvements to the אייוועלט forums.",
-  "version": "1.9.6",
+  "version": "1.9.7",
```

not sure if it will be a new version, 1.9.6 is still not published 
```suggestion
  "version": "1.9.6",
```